set.mm

```diff
@@ -457631,6 +457714,36 @@ Class abstractions (a.k.a. class builders)
       EBCRUIUOUEUBUFMUG $.
   $}
 
+  $( A set ` A ` not in a pair is neither element of the pair.  (Contributed by
+     Thierry Arnoux, 20-Nov-2023.) $)
+  nelpr $p |- ( A e. V -> ( -. A e. { B , C } <-> ( A =/= B /\ A =/= C ) ) )
+      $=
+    ( wcel cpr wn wceq wo wne wa elprg notbid neanior syl6bbr ) ADEZABCFEZGABHA
+    CHIZGABJACJKPQRABCDLMABACNO $.
+
+  ${
+    $d A x $.  $d B x $.  $d C x $.
+    $( Rewrite an empty intersection with a pair.  (Contributed by Thierry
+       Arnoux, 20-Nov-2023.) $)
+    inpr0 $p |- ( ( A i^i { B , C } ) = (/) <-> ( -. B e. A /\ -. C e. A ) ) $=
+      ( vx cv wne wa wral cpr cin c0 wceq wcel wn r19.26 wi wal wb 3bitr4i nelb
+      cvv nelpr elv imbi2i albii disj1 df-ral anbi12i ) DEZBFZUICFZGZDAHZUJDAHZ
+      UKDAHZGABCIZJKLZBAMNZCAMNZGUJUKDAOUIAMZUIUPMNZPZDQUTULPZDQUQUMVBVCDVAULUT
+      VAULRDUIBCUAUBUCUDUEDAUPUFULDAUGSURUNUSUODBATDCATUHS $.
+  $}
+
+  $( The class ` A ` is not in ` ( C \ { A , B } ) ` .  (Contributed by Thierry
```

Comment:
This is just the statement slightly rewritten. Maybe "An element of a pair is not an element of a difference with this pair."

Reply:
Yes! I was not very inspired.
I'll write "_The first element of a pair is not an element of a difference with this pair._" and similarly for the second such theorem.